mcs/class/System.Net.Http/corefx/Configuration.Certificates.cs

```diff
@@ -0,0 +1,117 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics;
+using System.IO;
+using System.Runtime.InteropServices;
+using System.Security.Authentication;
+using System.Security.Cryptography.X509Certificates;
+using System.Threading;
+using Xunit;
+
+namespace System.Net.Test.Common
+{
+    public static partial class Configuration
+    {
+        public static partial class Certificates
+        {
+            private const string CertificatePassword = "testcertificate";
+            private const string TestDataFolder = "TestData";
+
+            private static readonly Mutex m;
+            private const int MutexTimeout = 120 * 1000;
+
+            static Certificates()
+            {
+                if (PlatformDetection.IsUap)
```

Comment:
Seems to be copied from somewhere?

Reply:
It is, but the implementation has been modified.  The entire `GetResourceData()` method is new as well as our way of bundling the files as embedded resources.

They require the files to be available on disk and have some PowerShell script to do that.